langchain4j-core/src/main/java/dev/langchain4j/audit/api/event/AiServiceInvocationEvent.java

```diff
@@ -0,0 +1,55 @@
+package dev.langchain4j.audit.api.event;
+
+import dev.langchain4j.invocation.InvocationContext;
+
+public interface AiServiceInvocationEvent {
```

Comment:
```suggestion
public interface AiServiceEvent {
```
Would that still be clear?

Reply:
Done